ftw/noticeboard/types/category.py

```diff
@@ -0,0 +1,23 @@
+from ftw.noticeboard.interfaces import ICategory
+from plone.app.dexterity.behaviors.metadata import IBasic
+from plone.dexterity.content import Container
+from plone.supermodel import model
+from zope.interface import alsoProvides
+from zope.interface import implementer
+from zope.interface import implements
+from zope.schema.interfaces import ICollection
+
+
+@implementer(ICollection)
+class ICategorySchema(model.Schema):
+    """
+    This schema represents a category item.
+    """
+    pass
+
+
+alsoProvides(ICategorySchema, IBasic)
```

Comment:
What does this do?

Reply:
This implements the IBasic behaviour.

Reply:
But the behavior is added in the FTI (https://github.com/4teamwork/ftw.noticeboard/pull/2/files#diff-a1ca04c099d686312c1c53739bb70760R21) so this is not needed. We could provide the FormFields instead. See https://github.com/4teamwork/ftw.news/blob/cf2a4ecda4dc5f14f93c002d95b52d5b90ac1437/ftw/news/contents/news_folder.py#L16

Reply:
@2e12, any obejctions?